cdap-proto/src/main/java/io/cdap/cdap/proto/NamespaceMeta.java

```diff
@@ -188,6 +199,16 @@ public Builder setConfig(Map<String, String> configMap) {
       return this;
     }
 
+    public Builder setRepository(NamespaceRepositoryConfig repository) {
+      this.repository = repository;
+      return this;
+    }
+
+    public Builder deleteRepoConfig() {
```

Comment:
don't need this, can allow setting it to null with the setter (annotate NamespaceRepositoryConfig as `@Nullable`)